src/app/app.module.ts

```diff
@@ -31,11 +32,14 @@ import { SensorTypeComponent } from './form-controls/sensor-type/sensor-type.com
 import { SensorThemeComponent } from './form-controls/sensor-theme/sensor-theme.component';
 import { SensorStatusComponent } from './form-controls/sensor-status/sensor-status.component';
 import { SensorLocationComponent } from './form-controls/sensor-location/sensor-location.component';
+import { DataStreamComponent } from './form-controls/datastream/datastream.component';
```

Comment:
deze is nu dubbel, levert bij mij een error op

Reply:
Voor de rest werkt het top

Reply:
Oops, die had ik handmatig gemerged in de GitHub editor 🙈 Heb 'm gefixt!